### DISCUSSION TITLE
Is it possible to display in the week calendar not 7 days, but 3

### DISCUSSION THREAD

**Discussion by Denjs24** [nhn/tui.calendar#1335] [Q&A] [answered]
Hello, I would like to know if it is possible to display in the week calendar not 7 days, but 3 as in the screenshot. Found the field const WEEK_DAYS = 7; where 7 was replaced by 3, it would seem that what is needed, but no, it only shows the first 3 days of the week and scrolls through after 7 days. Please tell me how can I achieve the desired result?
![image](https://user-images.githubusercontent.com/63019948/208322285-245b48ac-6caf-415e-8041-e37789a5383f.png)

**Comment by adhrinae** [comment, marked as answer]
There's no option to change how many days are displayed in a week.

Perhaps you can check [this PR](https://github.com/nhn/tui.calendar/pull/1225) to modify it on your own.

Similar to #892.
